barman/clients/walrestore.py

```diff
@@ -241,21 +241,20 @@ def try_deliver_from_spool(config, dest_file):
     return otherwise.
 
     :param argparse.Namespace config: the configuration from command line
-    :param dest_file: The destination file object
+    :param dest_file: The path to the destination file
     """
-    spool_file = os.path.join(config.spool_dir, config.wal_name)
+    spool_file = str(os.path.join(config.spool_dir, config.wal_name))
```

Comment:
isn't the return value of `os.path.join` already a string?

Reply:
No, it's a `LiteralString`. Probably okay, but I try keeping the types correct.